src/main/webapp/resources/app/js/user/userService.js

```diff
@@ -1,22 +1,20 @@
 'use strict';
 define(['lodash'],
   function(_) {
-    var dependencies = ['$q', '$window', 'UserResource'];
-    var UserService = function($q, $window, UserResource) {
+    var dependencies = ['$window', 'UserResource'];
+    var UserService = function($window, UserResource) {
 
       function getLoginUser() {
-        if ($window.sessionStorage.getItem('user')) {
-          return $q.resolve(JSON.parse($window.sessionStorage.getItem('user')));
```

Comment:
I feel like removing caching entirely is a bit much; perhaps add a time to live to the cached item instead?